omod/src/main/compass/sass/patientdashboard/patientDashboard.scss

```diff
@@ -165,4 +165,49 @@ div {
 
 .report-value {
   color: lighten($highlight, 15%);
+}
+
+#visits-list {
+  border-left: 1px solid #cccccc !important;
+}
+
+#visits-list .viewVisitDetails{
```

Comment:
In the scss files we can use nested components. Better to use that to avoid unwanted css components and duplicates as well.
```
#visits-list {
 .viewVisitDetails {
   ...
  }
}
```

Reply:
done.

Reply:
@suthagar23 Any more feedback